src/org/labkey/targetedms/view/skylineAuditLogExtraInfoView.jsp

```diff
@@ -0,0 +1,20 @@
+<%@ page import="org.labkey.targetedms.TargetedMSController" %>
+<%@ page import="org.labkey.api.view.JspView" %>
+<%@ page import="org.labkey.api.view.HttpView" %>
+<%@ page import="org.labkey.targetedms.TargetedMSRun" %>
+<%@ page import="org.labkey.targetedms.parser.skyaudit.AuditLogEntry" %>
+<%@ page extends="org.labkey.api.jsp.JspBase" %>
+
+<%
+    JspView<TargetedMSController.SkylineAuditLogExtraInfoBean> me =
+            (JspView<TargetedMSController.SkylineAuditLogExtraInfoBean>) HttpView.currentView();
+    TargetedMSController.SkylineAuditLogExtraInfoBean bean = me.getModelBean();
+    AuditLogEntry entry = bean.getEntry();
+%>
+<div id="targetedmsAuditLogExtraInfo" >
+    <table>
+        <tr><td><strong>User Name:</strong> <%= h(entry.getUserName()) %></td><td><span class="fa fa-times">&nbsp;</span></td></tr>
+        <tr><td colspan="2"><strong>Entry Timestamp:</strong> <%=h(entry.getCreateTimestamp().toString())%></td></tr>
```

Comment:
Suggest using org.labkey.api.util.DateUtil.formatDateTime() instead as that will respect the date/time format as set by the server/project admin. I know that means translating to a java.util.Date

Reply:
Even easier: use formatDateTime() from JspBase. It will format the date using the current container's display format and encode it for you.